khmer/utils.py

```diff
@@ -33,4 +125,15 @@ def write_record(record, fp):
             '>{name}\n{seq}\n'.format(name=record.name,
                                       seq=record.sequence))
 
+
+def write_record_pair(read1, read2, fp):
```

Comment:
Pylint reminds us that 'fp' is both a) too short and b) not self documenting. 'file_object' or 'fileobj' would be clearer here.